### PR TITLE
example of how to use http.RedirectHandler and http.Redirect

### PR DESCRIPTION
**Pull request by markbates**

### CODE REVIEW

**Comment by markbates** [comment]
@ardan-bkennedy

File: topics/web/apis/example3/main.go [hunk 1]
```diff
@@ -24,7 +24,8 @@ func App() http.Handler {
 	r.Get("/customers/{id}", showHandler)
 	r.Get("/customers", indexHandler)
 	r.Post("/customers", createHandler)
-	r.Get("/", indexHandler)
+	// Redirect requests to / to /customers
```
Comment (ardan-bkennedy) [inline topics/web/apis/example3/main.go]: If you want this comment here are the rules.

A comment can't have anything on top of it.
A comment must have proper sentence structure and punctuation.

File: topics/web/apis/example3/main_test.go [hunk 2]
```diff
@@ -12,6 +12,28 @@ import (
 	"testing"
 )
 
+func TestRoot(t *testing.T) {
+
+	// Startup a server to handle processing these routes.
+	ts := httptest.NewServer(App())
+	defer ts.Close()
+
+	// Request the root URL
```
Comment (ardan-bkennedy) [inline topics/web/apis/example3/main_test.go]: Comment is missing sentence structure and punctuation.

File: topics/web/apis/example3/main_test.go [hunk 3]
```diff
@@ -12,6 +12,28 @@ import (
...
+	res, err := http.Get(ts.URL)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	// Find the URL we were redirected to:
```
Comment (ardan-bkennedy) [inline topics/web/apis/example3/main_test.go]: sentence structure and punctuation.  Consistency with other similar comments.

File: topics/web/apis/example4/main_test.go [hunk 4]
```diff
@@ -12,6 +12,28 @@ import (
 	"testing"
 )
 
+func TestRoot(t *testing.T) {
+
+	// Startup a server to handle processing these routes.
+	ts := httptest.NewServer(App())
+	defer ts.Close()
+
+	// Request the root URL
```
Comment (ardan-bkennedy) [inline topics/web/apis/example4/main_test.go]: sentence structure and punctuation.

File: topics/web/apis/example4/main_test.go [hunk 5]
```diff
@@ -12,6 +12,28 @@ import (
...
+	res, err := http.Get(ts.URL)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	// Find the URL we were redirected to:
```
Comment (ardan-bkennedy) [inline topics/web/apis/example4/main_test.go]: sentence structure and punctuation.

**Comment by markbates** [comment]
yeah, i figured you could do the commenting. :) i just threw a few in to be nice.

**Pull request merged**